shadow-cljs.edn

```diff
@@ -1,10 +1,11 @@
 {:deps true
  :builds {:app
           {:target :react-native
-           :init-fn app.core/start
+           :init-fn app.core/init
            :output-dir "app"
            :compiler-options {:source-map-path "app/"
                               :source-map-include-sources-content true
                               :warnings-as-errors true}
-           :devtools {:preloads [preload]
-                      :build-notify preload/build-notify}}}}
\ No newline at end of file
+           :devtools {:autoload true
```

Comment:
what's `:autoload`?

Reply:
> [`:autoload`](https://shadow-cljs.github.io/docs/UsersGuide.html#_lifecycle_hooks)
> A boolean controlling whether code should be hot loaded. Implicitly set to true if either of the callbacks is set. Always enabled for the :browser target by default, set to false to disable.

I took it from https://github.com/thheller/reagent-expo/blob/5886b98f1fbe5c173ede24520d72a76cebac2b98/shadow-cljs.edn#L15